CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ The types of changes are:
 - `Security` in case of vulnerabilities.
 
 ## [Unreleased](https://github.com/ethyca/fides/compare/2.32.0...main)
+- Added Erasure support for Iterable [#4695](https://github.com/ethyca/fides/pull/4695)
```

Comment:
Let's move this under the `Added` header, and lowercase `erasure`

Reply:
You got it!